frontend/app/(defaultLayout)/Troubleshooting/[device]/page.tsx

```diff
@@ -1,19 +1,23 @@
-import { flags } from '@config/flags';
 import { RestrictRobots } from '@helpers/next-helpers';
 import { getiFixitOrigin } from '@helpers/path-helpers';
 import { Metadata } from 'next';
 import { headers } from 'next/headers';
 import { notFound } from 'next/navigation';
-import TroubleshootingProblems, {
-   TroubleshootingProblemsProps,
-} from './components/troubleshootingProblems';
+import TroubleshootingProblems from './components/troubleshootingProblems';
+import { TroubleshootingProblemsApiData } from './hooks/useTroubleshootingProblemsProps';
 import {
    IFixitAPIClient,
    VarnishBypassHeader,
 } from '@ifixit/ifixit-api-client';
 
+export type AnswersProps = {
```

Comment:
We can omit `answers` from `TroubleshootingAnswersData` and use that as the type.

Reply:
https://github.com/iFixit/react-commerce/pull/2205/commits/120510c612a0e8a65d7b387ff4abfe0ac62319e5